### ISSUE TITLE
[Role handling] Required error message while adding non-existing members.  

### ISSUE THREAD

**Issue by varunmylaraiah** [freeipa/ansible-freeipa#411]
There is no error message while adding non-existing members(user/group/host/host group/service) in the role and also it removed existing member as well

```
[root@master ~]# ipa role-show newrole
  Role name: newrole
  Member users: vuser01
  Member groups: vgroup01
  Member hosts: teshoat.ipadomain.test
  Member host-groups: hostgroup01
  Privileges: User Administrators, Group Administrators, Host Administrators, DNS Administrators, DNS Servers
  Member services: newhost1/master.ipadomain.test@IPADOMAIN.TEST
```

```
[root@ansible ~]# cat newrole2.yaml
---
- name: "nonexisting user as a member"
  hosts: ipaserver

  tasks:
  - iparole:
      ipaadmin_password: <password>
      name: newrole
      user: nouser
      action: member

[root@ansible ~]# ansible-playbook -vv -i inventory/server.hosts newrole2.yaml 
ansible-playbook 2.9.11
  config file = /root/ansible.cfg
  configured module search path = ['/root/ansible-freeipa/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.8/site-packages/ansible
  executable location = /usr/bin/ansible-playbook
  python version = 3.8.5 (default, Aug 12 2020, 00:00:00) [GCC 10.2.1 20200723 (Red Hat 10.2.1-1)]
Using /root/ansible.cfg as config file

PLAYBOOK: newrole2.yaml **********************************************************************************************
1 plays in newrole2.yaml

PLAY [nonexisting user as a member] **********************************************************************************

TASK [Gathering Facts] ***********************************************************************************************
task path: /root/newrole2.yaml:2
[DEPRECATION WARNING]: Distribution fedora 32 on host master.ipadomain.test should use /usr/bin/python3, but is using
 /usr/bin/python for backward compatibility with prior Ansible releases. A future Ansible release will default to 
using the discovered platform python for this host. See 
https://docs.ansible.com/ansible/2.9/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False 
in ansible.cfg.
ok: [master.ipadomain.test]
META: ran handlers

TASK [iparole] *******************************************************************************************************
task path: /root/newrole2.yaml:6
changed: [master.ipadomain.test] => {"changed": true}
META: ran handlers
META: ran handlers

PLAY RECAP ***********************************************************************************************************
master.ipadomain.test      : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0 
```

```

[root@master ~]# ipa role-show newrole
  Role name: newrole
  Member groups: vgroup01
  Member hosts: teshoat.ipadomain.test
  Member host-groups: hostgroup01
  Privileges: User Administrators, Group Administrators, Host Administrators, DNS Administrators, DNS Servers
```

Expected:
Should not remove existing members and should display an error if entry not exists.

CLI Console output:
```
[root@master ~]# ipa role-add-member newrole --users=nouser 
  Role name: newrole
  Member groups: vgroup01
  Member hosts: teshoat.ipadomain.test
  Member host-groups: hostgroup01
  Privileges: User Administrators, Group Administrators, Host Administrators, DNS Administrators, DNS Servers
  Failed members: 
    member user: nouser: no such entry
    member group: 
    member host: 
    member host group: 
    member service: 
    member User ID override: 
-------------------------
Number of members added 0
-------------------------
```

**Comment by varunmylaraiah** [comment]
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1893679

**Comment by rjeffman** [comment]
Closed by #469.
